### DISCUSSION TITLE
Cannot Deploy Data Science Environment

### DISCUSSION THREAD

**Discussion by gigijinglew** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#71] [3.0 VMs and data (Azure VM, local deployment, and Jupyter)]
I am trying to create a custom deployment following the **1b: Getting Started: Azure Environment.** I am stuck on deploying data science environment as shown in the 1st screenshot. I followed every step in the instruction but it says:

The template deployment failed with error: 'The resource with id: '/subscriptions/41eb2748-7ed3-4d97-8775-69d742b2b370/resourceGroups/cubeinabox/providers/Microsoft.Compute/virtualMachines/CubeInABox-CPU-16GB' failed validation with message: 'The requested size for resource '/subscriptions/41eb2748-7ed3-4d97-8775-69d742b2b370/resourceGroups/cubeinabox/providers/Microsoft.Compute/virtualMachines/CubeInABox-CPU-16GB' is currently not available in location 'australiaeast' zones '' for subscription '41eb2748-7ed3-4d97-8775-69d742b2b370'. Please try another size or deploy to a different location or zones. See https://aka.ms/azureskunotavailable for details.'.'. (Code: InvalidTemplateDeployment)

I checked #42 but actually, my subscription is Azure for Students and I do not have any other vm existing. So, I cannot request to increase a quota. 
![截屏2021-05-24 下午2 30 39](https://user-images.githubusercontent.com/84652101/119307339-2534bd00-bc9e-11eb-9bec-ad3544394468.png)
![截屏2021-05-24 下午2 35 17](https://user-images.githubusercontent.com/84652101/119307351-2a920780-bc9e-11eb-9dde-395367c5cbf2.png)

Does anyone have any idea? Thank you so much!

**Comment by EY-glowglowglow** [comment]
Hi there, can you please try deploying in Australia Southeast? That has solved the issue for other participants with the same error. :)

**Reply by gigijinglew** [reply]
Thank you so much. It works!
